telemetry.go

```diff
@@ -0,0 +1,32 @@
+package gocron
```

Comment:
it'd be cool to think about what other metics users would benefit from.

some i can think of:
- count of runs by job
- for the singleton / waitmode I could see having a metric for how many jobs are waiting / queued - could do this by incrementing a metric when one is queued and decrementing when it's run